hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/infrastructure/audit/LoggingAuditorSpec.groovy

```diff
@@ -0,0 +1,80 @@
+package pl.allegro.tech.hermes.management.infrastructure.audit
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.javers.core.Javers
+import org.javers.core.JaversBuilder
+import org.junit.Before
+import org.junit.BeforeClass
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import pl.allegro.tech.hermes.api.Group
+import pl.allegro.tech.hermes.management.utils.MockAppender
+import spock.lang.Specification
+
+import static org.slf4j.Logger.ROOT_LOGGER_NAME
+
+class LoggingAuditorSpec extends Specification {
+
+    static final String TEST_USER = "testUser"
+    static final Group TEST_GROUP = new Group("testGroup", "testOwner", "testSupportTeam", "testContact")
+
+    static Logger root = LoggerFactory.getLogger(ROOT_LOGGER_NAME)
```

Comment:
These shoudn't be statics. If they should be shared across tests, there's a `@Shared` annotation. Here I think there's no issue with recreating them for each test.